OpenRA.Mods.Common/Orders/GlobalButtonOrderGenerator.cs

```diff
@@ -42,6 +42,7 @@ protected IEnumerable<Order> OrderInner(World world, MouseInput mi)
 		{
 			if (mi.Button == MouseButton.Left)
 			{
+				// TODO: Should this extend to controlling allied units?
```

Comment:
I would say yes. it is a bit irritating that allied units not response when selected